### PR TITLE
Merge edge-server-template feature branch into main 

### PR DESCRIPTION
**Pull request by cbullinger**
Add new `other/edge-server` directory containing client and backend files for new Edge Server template app

### CODE REVIEW

File: other/edge-server/client/node-server/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+TODO: Write info about this server and how it fits into the Edge Server template
```
Comment (dacharyc) [inline other/edge-server/client/node-server/README.md:1]: Oops. We should probably either populate details here or remove this README.md from the branch.

File: other/edge-server/client/node-server/dist/js/db/connect.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        function step(result) { result.done ? resolve(result.value) : adopt(result.value).then(fulfilled, rejected); }
+        step((generator = generator.apply(thisArg, _arguments || [])).next());
+    });
+};
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.getTodoCollection = exports.connectToEdgeServer = void 0;
+const mongodb_1 = require("mongodb");
+const uri = process.env.EDGE_SERVER_URI;
+// const uri = "mongodb://localhost:27021";
+console.log(uri);
```
Comment (dacharyc) [inline other/edge-server/client/node-server/dist/js/db/connect.js:16]: Should we remove these two lines from this file?

File: other/edge-server/client/react-client/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+## Edge Server React Client
+
+TODO: add info about how this client fits into the Edge Server template
```
Comment (dacharyc) [inline other/edge-server/client/react-client/README.md:3]: Same Q as above. We should probably either populate details here or remove this README.md from the branch.

File: other/edge-server/client/react-client/index.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    />
+    <meta
+      name="theme-color"
+      content="#000000"
+    />
+    <meta
+      name="description"
+      content="A Todo List App built on MongoDB Atlas."
```
Comment (dacharyc) [inline other/edge-server/client/react-client/index.html:19]: Should this say something about Edge Server?

**Review by dacharyc — CHANGES_REQUESTED** [review]
I think we need to do something with the README files in the React client and Node.js server directories - either remove them or add more details. There are a couple of other small non-blocking Qs but I think those READMEs are blocking.

File: other/edge-server/client/node-server/dist/js/db/connect.js (hunk 2 above)
Comment (krollins-mdb) [reply other/edge-server/client/node-server/dist/js/db/connect.js:16]: Yep! Please remove.

File: other/edge-server/client/react-client/index.html (hunk 4 above)
Comment (krollins-mdb) [reply other/edge-server/client/react-client/index.html:19]: Yeah, something like this, maybe?
```suggestion
      content="A Todo List App built with Atlas Edge Server."
```

**Review by dacharyc — APPROVED** [review]
LGTM! Nice teamwork, y'all! 🎉  ✋

**Pull request merged**